news/2019-03-31-beatmap-spotlights-winter-2018.md

```diff
@@ -0,0 +1,308 @@
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/iron-lung.jpg)](https://osu.ppy.sh/beatmapsets/916059)
+
+[Myxomatosis](https://osu.ppy.sh/users/2202645) put his spin on Radiohead's "My Iron Lung", an alternative rock classic which finally has it's chance to see the light of day on osu. Myxo uses challenging yet intuitive movement concepts making the map exciting till the end and players yearning for more!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/gin.jpg)](https://osu.ppy.sh/beatmapsets/902948)
+
+[rrtyui](https://osu.ppy.sh/users/352328) brings us a new test of skill on "Gin no Kaze" by wa. remixed celas, sparking a new life into the ideas of Fear's version. As the song slowly reveals itself, the map follows suit to prepare the player for its climax. There, the small circle size is pushed to its limits with an incessant series of unusually precise slider motions. 
+
+Even though hard, this map is also subtle. Combining clever mapping decisions worthy of an essay, rrtyui shows that they have a deep understanding of all layers in mapping. Those who need to improve a bit more to take on the highest difficulty can definitely find a similar challenge in [Uberzolik](https://osu.ppy.sh/users/1314547)'s another.
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/first-town.jpg)](https://osu.ppy.sh/beatmapsets/888513)
+
+Staying true to the chiptune-y video game style of Camellia's First Town Of This Journey, [ScubDomino](https://osu.ppy.sh/users/8972308) plasters densely syncopated slider patterning on a beautifully scenic 8-bit background. Every difficulty, featuring guest mappers [Milan-](https://osu.ppy.sh/users/1052994), [Sing](https://osu.ppy.sh/users/3795679), and [Delis](https://osu.ppy.sh/users/1603923), follows similar conventions, using sliderstreams and overlaid sliders to give the entire mapset a cohesive motif culminating in a spectacular interpretation through ScubDomino's [Adventure] difficulty. 
+
+The map itself follows its own journey, beginning with highly grid-oriented patterning that slowly diverges from these constraints in a magnificent climax rich with sliderstreams and velocity changes that slowly reverts back to its more structured composition as the music also returns home. Find yourself grooving along to this bright, upbeat tune as you embark on your personal journey through this mapset.
```

Comment:
"ScubDomino plasters densely syncopated slider patterning **on a beautifully scenic 8-bit background he drew himself.**"

"in a magnificent climax rich with sliderstreams and velocity changes **and finally reverts back to** its more structured composition"

Just wanting to make a couple edits to the description I wrote.

Reply:
Use this:

![](https://puu.sh/D7gd5/3c18af5746.png)

Reply:
Should I use that too if the mappers in the graphic for First Town of this Journey aren't capitalized?

Reply:
You can only change text with that, so a regular comment is fine.